File: CodeForces/A_Chewbaсca_and_Number.py
s = input()
ans = ""
for i in range(len(s)):
    if s[0] == 9:
        break
    a = int(s[i])
    b = 9 - a
    if a > b:
        ans += str(b)
    elif a < b:
        ans += str(a)
if s[0] == 9:
    ans = 9
print(ans)

File: CodeForces/A_Chewbaсca_and_Number.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    string s, ans = " "; cin >> s;
    for(int i = 0; i < s.length(); i++){
        int a = (int) s[i];
        int b = 9 - a;
        if(a > b) ans += (char) b;
        else if(a < b) ans += (char) (a);
    }
    cout << ans << endl;
}